src/server/env/index.js

```diff
@@ -1,15 +1,13 @@
 /*
- * This file sets up the environment variables based on the value of `process.env.APP_ENV`.
+ * This file sets up the environment variables based on the value of `process.env.ENV_FILE`.
  * These variables are needed at both bundle time and at run time. The variables should be
  * set up at the entry point of the application prior to bundling.
+ * Also written in ES5 due to need to be used in webpack.
```

Comment:
Is this still used in both webpack and the application? I'd prefer there was an inline version for webpack, & this file remained application-only. The `ENV_FILE` behaviour introduces complexity for any devs attempting to understand it, & it's outright confusing having webpack load up a file that's under `/src/server`. 😕